WinUIGallery/ControlPages/SystemBackdropsPage.xaml

```diff
@@ -19,14 +19,19 @@
       mc:Ignorable="d">
 
     <StackPanel>
+        <local:ControlExample HeaderText="Create a window with a built-in Mica system backdrop."
```

Comment:
@karkarl I'd include a XAML snippet here as well now that's fully supported? It's probably how most devs would (like) to use it?

Reply:
What should be in the Xaml snippet? I can't think of anything relevant to mica implementation to put in there.

Reply:
```XAML
<Window.SystemBackdrop>
     <MicaBackdrop/>
</Window.SystemBackdrop>
```

and

```XAML
<Window.SystemBackdrop>
    <DesktopAcrylicBackdrop/>
</Window.SystemBackdrop>
```

Reply:
Added!